BUILD.md

```diff
@@ -226,9 +226,10 @@ It fixes some source files to add missing `#include`s.
 
     ```
     conan install .. --output-folder . --build missing --settings build_type=Release
-    conan install .. --output-folder . --build missing --settings build_type=Debug
     ```
 
+    To build Debug, set `build_type=Debug` (instead of Release) and in the next step, set `-DCMAKE_BUILD_TYPE=Debug`
+
```

Comment:
How about including both `conan install` commands?
```
conan install .. --output-folder . --build missing --settings build_type=Release
conan install .. --output-folder . --build missing --settings build_type=Debug
```
or even nerdier
```
for type in Release Debug
do
  conan install .. --output-folder . --build missing --settings build_type=${type}
done
```

Most developers should be able to build rippled with either setting, and this has them ready for both. If they don't want both, it's more obvious (IMHO) to leave a command out than it is to add a second one if they do. Worst case, they waste the extra time and disk space for something they don't need, but the time is only wasted once.

Reply:
ok. I moved this line up, since it's what I missed the first time.
```
    Pass the CMake variable [`CMAKE_BUILD_TYPE`][build_type]
    and make sure it matches the `build_type` setting you chose in the previous
    step.
```